src/display_manager.js

```diff
@@ -0,0 +1,40 @@
+export default class DisplayManager {
+  static reset = (element) => {
+    element.innerHTML = '';
+  };
+
+  static displayTask = (parent, task) => {
+    // console.log(parent, task);
```

Comment:
- [ ] [OPTIONAL] please remove any commented out code, this will make your codebase look a lot cleaner and more readable.